cpp/oneapi/dal/backend/primitives/reducer_dpc_test.cpp

```diff
@@ -0,0 +1,171 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <CL/sycl.hpp>
+
+#include "gtest/gtest.h"
+
+#define ONEAPI_DAL_DATA_PARALLEL
+
+#include "oneapi/dal/backend/primitives/common.hpp"
+#include "oneapi/dal/backend/primitives/reducer.hpp"
+#include "oneapi/dal/data/array.hpp"
+
+TEST(reducer_l2_rm, can_handle_array_of_zeros) {
+    cl::sycl::queue q{ cl::sycl::gpu_selector() };
```

Comment:
Don't create queue in the test. Use `DECLARE_TEST_POLICY` and see how it's done in tests for other primitives